@xen-orchestra/proxy/src/app/mixins/backups/_deduped.js

```diff
@@ -0,0 +1,46 @@
+import ensureArray from 'ensure-array'
+import Resource from 'promise-toolbox/_Resource'
+import { MultiKeyMap } from '@vates/multi-key-map'
+
+export const debounceResource = (resource, delay) => {
+  const expires = Date.now() + delay
```

Comment:
This is not what we want, we simply wants to delay the disposer for a given duration after it has been called.